test-all.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+echo "Testing all kata setup scripts"
+find -s . -name "setup.sh" -type f -execdir ../utils/test/test_setup.sh  \;
```

Comment:
`-s` is not recognized on my setup. When I delete it, I get a list of `OK` for the katas. Can you post a snippet from the man-page on your machine of what `-s` is supposed to do?

Reply:
```
-s      Cause find to traverse the file hierarchies in lexicographical order, i.e., alphabetical order within each directory.  Note: `find
             -s' and `find | sort' may give different results.
```